docs/index.yaml

```diff
@@ -1,9 +1,18 @@
 apiVersion: v1
 entries:
   wg-access-server:
+  - apiVersion: v2
+    appVersion: v0.5.1
+    created: "2022-01-26T08:53:44.468726774Z"
+    description: A Wireguard VPN Access Server
+    digest: 2b3a8bbad6eaeffef433fe0bb0421b8005a3083abf1111afc898b61123749ad0
+    name: wg-access-server
+    urls:
+    - https://freie-netze.org/wg-access-server/charts/wg-access-server-v0.5.1.tgz
```

Comment:
here i got a 404

Reply:
That's to be expected as github pages doesn't roll out non-master branches

Reply:
ok i thought because the Version is already as release there: https://github.com/freifunkMUC/wg-access-server/releases

Reply:
Yeah, but we haven't bumped the chart version yet. That's why you're getting a 404